build/source/engine/bigAquifer.f90

```diff
@@ -108,7 +108,7 @@ subroutine bigAquifer(&
  scalarAquiferBaseflow = aquiferBaseflowRate*(xTemp**aquiferBaseflowExp)
 
  ! compute the derivative in the net aquifer flux
- dBaseflow_dAquifer    = -(aquiferBaseflowExp*aquiferBaseflowRate*(xTemp**(aquiferBaseflowExp - 1_i4b)))/aquiferScaleFactor ! aquiferBaseflowExp is integer valued
+ dBaseflow_dAquifer    = -(aquiferBaseflowExp*aquiferBaseflowRate*(xTemp**(aquiferBaseflowExp - 1._rkind)))/aquiferScaleFactor
```

Comment:
Why does this revert the initial change from `rkind` to `i4b`?

This one is the first such a reverted change that I see. Are there more?

Reply:
Both `aquiferBaseflowExp - 1_i4b` and  `aquiferBaseflowExp - 1._rkind)` will produce the same numerical value because of Fortran's mixed mode arithmetic rules (e.g., https://fortranwiki.org/fortran/show/Arithmetic or https://docs.oracle.com/cd/E19957-01/805-4939/z400073a2265/index.html). Given that `aquiferBaseflowExp` is of type `rkind`, the `1_i4b` integer literal is automatically promoted to type `rkind` (which has the most storage units of the two data types).

However, after some thought, it seemed  most logical to preserve the original  `aquiferBaseflowExp - 1._rkind)` form because that correctly implies that `aquiferBaseflowExp` is of type `rkind`. I thought that this form would be clearer for developers, hence the revision.